openpype/hosts/traypublisher/csv_publish.py

```diff
@@ -0,0 +1,85 @@
+import os
+
+import pyblish.api
+import pyblish.util
+
+from openpype.client import get_asset_by_name
+from openpype.lib.attribute_definitions import FileDefItem
+from openpype.pipeline import install_host
+from openpype.pipeline.create import CreateContext
+
+from openpype.hosts.traypublisher.api import TrayPublisherHost
+
+
+def csvpublish(
+    csv_filepath,
+    project_name,
+    asset_name,
+    task_name=None,
+    username=None,
+    targets=None
```

Comment:
Shouldn't the targets be rather defined by the function itself?

Reply:
Yeah this is better not be part of cli. Addressed via e4fa43c86e495c6da6e448969fc0067a8bfa42a7